src/Providers/Trustpilot/Request/AccessTokenRequest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace PodPoint\Reviews\Providers\Trustpilot\Request;
+
+use GuzzleHttp\Psr7\Request;
+use PodPoint\Reviews\AccessToken;
+use PodPoint\Reviews\Request\BaseRequest;
+
+/**
+ * Class AccessTokenRequest
+ * @package PodPoint\Reviews\Providers\Trustpilot\Request
+ */
+class AccessTokenRequest extends BaseRequest
+{
+    const API_KEY = 'apiKey';
+    const API_SECRET = 'apiSecret';
+    const USERNAME = 'username';
+    const PASSWORD = 'password';
+
+    public function requiredFields(): array
```

Comment:
I think I would make this a class constant array that uses the four class constants above